commerce-cloud-code/app_yourshophere/cartridge/partials/order/orderitems.js

```diff
@@ -43,10 +43,22 @@ exports.createModel = function createModel(options) {
                 url: `${image.url}?${options.settings.imageDISConfig}`,
                 alt: image.alt,
             })),
-            pdpUrl: URLUtils.url('Product-Show', 'pid', item.productID).toString(),
+            pdpUrl: options.settings.linkProducts !== false ? URLUtils.url('Product-Show', 'pid', item.productID).toString() : null,
+            analytics: JSON.stringify({
+                enrichmentTypes: [
```

Comment:
I'd call it `contribution` or `contributesTo` and then just have the partial event which we will merge via `Object.assign()`ish